code/modules/library/bibles.dm

```diff
@@ -0,0 +1,353 @@
+GLOBAL_LIST_INIT(biblenames, list(
+	"Bible",
+	"Quran",
+	"Scrapbook",
+	"Burning Bible",
+	"Clown Bible",
+	"Banana Bible",
+	"Creeper Bible",
+	"White Bible",
+	"Holy Light",
+	"The God Delusion",
+	"Tome",
+	"The King in Yellow",
+	"Ithaqua",
+	"Scientology",
+	"Melted Bible",
+	"Necronomicon",
+	"Insulationism",
+	"Guru Granth Sahib",
+	"Kojiki",
+))
+//If you get these two lists not matching in size, there will be runtimes and I will hurt you in ways you couldn't even begin to imagine
+// if your bible has no custom itemstate, use one of the existing ones
+GLOBAL_LIST_INIT(biblestates, list(
+	"bible",
+	"koran",
+	"scrapbook",
+	"burning",
+	"honk1",
+	"honk2",
+	"creeper",
+	"white",
+	"holylight",
+	"atheist",
+	"tome",
+	"kingyellow",
+	"ithaqua",
+	"scientology",
+	"melted",
+	"necronomicon",
+	"insuls",
+	"gurugranthsahib",
+	"kojiki",
+))
+GLOBAL_LIST_INIT(bibleitemstates, list(
+	"bible",
+	"koran",
+	"scrapbook",
+	"burning",
+	"honk1",
+	"honk2",
+	"creeper",
+	"white",
+	"holylight",
+	"atheist",
+	"tome",
+	"kingyellow",
+	"ithaqua",
+	"scientology",
+	"melted",
+	"necronomicon",
+	"kingyellow",
+	"gurugranthsahib",
+	"kojiki",
+))
+
+/obj/item/book/bible
+	name = "bible"
+	desc = "Apply to head repeatedly."
+	icon = 'icons/obj/storage/book.dmi'
+	icon_state = "bible"
+	worn_icon_state = "bible"
+	inhand_icon_state = "bible"
+	lefthand_file = 'icons/mob/inhands/items/books_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items/books_righthand.dmi'
+	force_string = "holy"
+	unique = TRUE
+	/// Deity this bible is related to
+	var/deity_name = "Space Jesus"
+
+/obj/item/book/bible/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/anti_magic, MAGIC_RESISTANCE_HOLY)
+	carve_out()
+
+/obj/item/book/bible/examine(mob/user)
+	. = ..()
+	if(deity_name)
+		. += span_notice("This bible has been approved by [deity_name].")
+	if(user?.mind?.holy_role)
```

Comment:
when the flarp is user gonna be null here? if you know it's going to runtime 100% of the time, i'd rather an explicit early return than the `?` because this will obscure any potential runtime weirdness that should be fixed instead.

otherwise:

```suggestion
	if(user.mind?.holy_role)
```

this probably isn't you that added this in and is just a consequence of the copypasta refactor but man it makes my blood boil